### PR TITLE
journal

### PR DESCRIPTION
**Pull request by symbiont-stevan-andjelkovic**
- docs(course): add motivation for each chapter
- feat(journal): add new failing test

### CODE REVIEW

File: doc/advanced-testing-mini-course/README.md [hunk 1]
```diff
@@ -83,22 +92,96 @@ step m cmd = case cmd of
...
+2. Collect timing information about how long each command takes to execute on
+   average.
+
+* See also
+
+  - Why state machines over other forms of specifications?
+    + Executable (we will use this later)
+    + Gurevich's generalisation of the Church-Turing thesis
+    + Already heavily used in distributed systems
+
```
Comment (symbiont-daniel-gustafsson) [inline doc/advanced-testing-mini-course/README.md:118]: We should probably also have an exercise of
-  finding a bug
-  make a regression test
-  fix bug
To make sure that we teach early about importance of regression tests.

**Review by symbiont-daniel-gustafsson — APPROVED** [review]

File: doc/advanced-testing-mini-course/README.md (hunk 1 above)
Comment (symbiont-stevan-andjelkovic) [reply doc/advanced-testing-mini-course/README.md:131]: Would be nice if we did all that together while live coding for some contrived bug, but then left an other bug as exercise (that perhaps needs generator tweaking to appear, or at least bigger `--quickcheck-size`)...

Comment (symbiont-stevan-andjelkovic) [reply doc/advanced-testing-mini-course/README.md:131]: Or perhaps the exercise should be to test a completely different SUT?

Comment (symbiont-daniel-gustafsson) [reply doc/advanced-testing-mini-course/README.md:131]: Yeah the exercise might be a different SUT

**Pull request merged**